app/views/layouts/_navigation.html.erb

```diff
@@ -36,6 +36,9 @@
               <% if can?(:read, ToolType) %>
                 <li><%= link_to "Tool Types", tool_types_path %></li>
               <% end %>
+              <% if is_admin? %>
```

Comment:
I originally did this because there is no Report model to authorize, but after some googling I learned you can authorize symbols. So I'll change this to use cancancan via can?(:read, :reports). Also the method in ApplicationController will be changed to use cancancan but it still needs to be there since Dossier needs a reference method to call (since it automatically creates controllers for these reports).